webui/react/src/services/decoder.ts

```diff
@@ -165,7 +166,7 @@ export const jsonToExperiment = (data: unknown): Experiment => {
   return {
     archived: io.archived,
     config: ioToExperimentConfig(io.config),
-    configRaw: (data as { config: Record<string, unknown> }).config,
+    configRaw: testConfig, //(data as { config: Record<string, unknown> }).config,
```

Comment:
**question:** was this supposed to get restored to what it was before? I'm guessing this was just for testing purposes?

Reply:
yep 
> TODO: remove the tmp commit (decoder override)